index.js

```diff
@@ -6,6 +6,85 @@ class Subsume {
 	static parse(str, id) {
 		return (new Subsume(id)).parse(str);
 	}
+
+	static parseAll(str, ids) {
+		if (ids && ids.constructor !== Array) {
```

Comment:
This is a weird way to check for an array. The correct way is to use `Array.isArray()`.